Rakefile

```diff
@@ -18,7 +18,7 @@ require_relative 'tasks/assets'
 require_relative 'tasks/profiles'
 require_relative 'tasks/geo'
 
-Mongoid.load!('./config/mongoid.yml')
+Mongoid.load!('./config/mongoid.yml', :test)
```

Comment:
The appropriate ENV variable is set inside `test/test_helper.rb`. See line 1 `ENV['RACK_ENV'] = 'test'`. Should work as expected. Have you tried `bundle exec rake test`?